frontend/pages/index.tsx

```diff
@@ -92,6 +100,31 @@ export default function Home(): JSX.Element {
           );
         })}
       </div>
+
+      <Flex direction="column" align="center" width="lg">
```

Comment:
Could you set the height of this to "100vw"? Apologies if it wasn't clear from the mockup, but the subsection should take up the entire page height (similar vibe as https://chakra-ui.com/)